plugins/search-react/package.json

```diff
@@ -51,6 +52,7 @@
     "@backstage/cli": "^0.19.0-next.2",
     "@backstage/core-app-api": "^1.1.0-next.2",
     "@backstage/test-utils": "^1.2.0-next.2",
+    "@internal/plugin-catalog-customized": "0.0.2-next.0",
```

Comment:
should remove this, right

Reply:
It was supposed to be `@backstage/plugin-catalog`, thanks for pointing it out!
Is it ok to have another plugin as a development dependency? I'm using `CatalogSearchResultListItem` in one of the stories.

Reply:
No, unfortunately not. Plugins and -react packages can depend on -react packages, but not on plugins. Could the story not be in the other package?

Reply:
Good question... Do you have any suggestions for another package, @emmaindal?

Reply:
can we just use the Default result list item instead for the storybook? https://github.com/backstage/backstage/blob/master/plugins/search-react/src/components/DefaultResultListItem/DefaultResultListItem.tsx

Reply:
I added a simplified version of the `CatalogSearchResultListItem` and removed the catalog plugin as a dependency 🙌🏻